cdap-common/src/main/java/co/cask/cdap/common/HttpExceptionHandler.java

```diff
@@ -34,12 +34,16 @@
   @Override
   public void handle(Throwable t, HttpRequest request, HttpResponder responder) {
     if (t instanceof BadRequestException) {
+      logWarnings(request, t);
```

Comment:
Yes you are correct. But isn't it required to log it so that we in a typical case of bad request or not found or something we can look into the logs as what was being requested etc and also the stack trace to debug it ?